cookbook/entities/creating_entities.rst

```diff
@@ -0,0 +1,191 @@
+How to Create Entities
+======================
+
+When working with the OroPlatform, creating entities and map them to the database is not different
+from doing the same in a common Symfony application. After you have created your bundle, create the
+entity classes you need in the bundle's ``Entity`` namespace, add all the needed properties, and
+add the needed mapping annotations as usual.
+
+A task is composed of a brief subject, a more verbose description, a due date, and a priority.
+Also, each task is identified by a unique identifier that will be automatically generated by the
+database:
+
+.. code-block:: php
+    :linenos:
+
+    // src/AppBundle/Entity/Task.php
+    namespace AppBundle\Entity;
+
+    use Doctrine\ORM\Mapping as ORM;
+
+    /**
+     * @ORM\Entity()
+     * @ORM\Table(name="app_task")
+     */
+    class Task
+    {
+        /**
+         * @ORM\Id()
+         * @ORM\GeneratedValue(strategy="AUTO")
+         * @ORM\Column(type="integer")
+         *
+         * @var int
+         */
+        private $id;
+
+        /**
+         * @ORM\Column(type="string")
+         *
+         * @var string
+         */
+        private $subject;
+
+        /**
+         * @ORM\Column(type="text")
+         *
+         * @var string
+         */
+        private $description;
+
+        /**
+         * @ORM\Column(type="datetime", name="due_date")
+         *
+         * @var \DateTime
+         */
+        private $dueDate;
+
+        /**
+         * @ORM\ManyToOne(targetEntity="Priority")
+         * @ORM\JoinColumn(name="task_priority_id", onDelete="SET NULL")
+         *
+         * @var Priority
+         */
+        private $priority;
+
+        public function getId()
+        {
+            return $this->id;
+        }
+
+        public function getSubject()
+        {
+            return $this->subject;
+        }
+
+        public function setSubject($subject)
+        {
+            $this->subject = $subject;
+        }
+
+        public function getDescription()
+        {
+            return $this->description;
+        }
+
+        public function setDescription($description)
+        {
+            $this->description = $description;
+        }
+
+        public function getDueDate()
+        {
+            return $this->dueDate;
+        }
+
+        public function setDueDate(\DateTime $dueDate)
+        {
+            $this->dueDate = $dueDate;
+        }
+
+        public function getPriority()
+        {
+            return $this->priority;
+        }
+
+        public function setPriority(Priority $priority)
+        {
+            $this->priority = $priority;
+        }
+    }
+
+Priorities for tasks will be configurable. Thus, they are modeled as separate entities:
```

Comment:
Is `configurable` in terms of ORO? In this case I does not see any Config annotation in `Priority ` entity definition.